### DISCUSSION TITLE
How can I set different themes for the light/dark theme.

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#6789] [Q&A]
When I choose the light/dark theme in the LO impress, it doesn't change its appearance to light ones. How can I set different theme for the light theme? Here's the [demo](https://files.catbox.moe/562bky.mp4). 
P.S.: LO - LibreOffice

**Comment by Misnad** [comment]
Same here. I'm stuck on the light theme.
